packages/server/src/rateLimit/index.ts

```diff
@@ -51,8 +52,13 @@ export default class RateLimit {
         return false;
       }
 
+
+      const requestIdPrefix = requestId ? formatRequestIdMessage(requestId) : '';
+      this.logger.warn(`${requestIdPrefix}, Rate limit call to ${methodName}, ${this.database[ip].methodInfo[methodName].remaining} out of ${total} calls remaining`);
```

Comment:
This double the logs in this flow.
I also think it's a duplicate.
Just use the original.
Honestly I forget why we check for a valid requestId. In deployed flows it will be valid. If it's missing an if in test case it's probably not a big deal